stories/Stepper/HeaderVariants.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import {
+  StepperContainer,
+  StepperHeader,
+  StepperHeaderElement,
+  Icon
+} from '../../src'
+const HeaderVariants = () => {
+  return (
+    <StepperContainer dark>
+      <StepperHeader>
+        <StepperHeaderElement variant="confirmed" icon="it-check">
+          Label Step 1
+        </StepperHeaderElement>
+        <StepperHeaderElement variant="active">
+          Label Step 2
+        </StepperHeaderElement>
+        <StepperHeaderElement>Label Step 3</StepperHeaderElement>
+        <StepperHeaderElement variant="steppers-index">
+          2/6
+        </StepperHeaderElement>
+      </StepperHeader>
+      [` `]
+      <StepperHeader>
+        <StepperHeaderElement
+          variant="confirmed"
+          icon="it-check"
+          iconName="it-calendar">
+          Label Step 1
+        </StepperHeaderElement>
+        <StepperHeaderElement variant="active" iconName="it-lock">
+          Label Step 2
+        </StepperHeaderElement>
+        <StepperHeaderElement iconName="it-settings">
+          Label Step 3
+        </StepperHeaderElement>
+        <StepperHeaderElement variant="steppers-index">
+          2/6
+        </StepperHeaderElement>
+      </StepperHeader>
+      [` `]
```

Comment:
Why are these square brackets here?

Reply:
Oh ! added these just for formatting. It adds some space between the examples.

Reply:
I think it'll be okay without it too, also were you trying to use `{" "}`?

Reply:
I don't know the difference between usage of `[` `]` and `{" "}` but the previous seemed to add some space. `{" "}` doesn't make any difference. However, I should probably get rid of it.